vscode/src/extension.ts

```diff
@@ -34,6 +35,7 @@ export function activate(context: vscode.ExtensionContext) {
         }
       });
   }
+  registerDiffView(context);
```

Comment:
This illustrates what I believe to be my problem with this PR as currently constructed. It takes our extension down a completely different path with a whole new tree view and 8 new commands that are not readily tied into the work that @ibolton336 has done.

As an example, in this registerDiffView you define the command for `konveyor.diffView.applyAll`, so this PR would necessitate an entirely new command to be created for fixes to be accepted from the webview. This is not the direction I believe we should be heading in.

Reply:
IMHO it fits our roadmap. 

As for `konveyor.diffView.applyAll`,  you are right - the command should be in the global namespace i.e. `konveyor.applyAll`

Reply:
1. Moved registration into `VsCodeExtension` following the example in https://github.com/konveyor/editor-extensions/pull/90 (by @ibolton336 ).
2. renamed `konveyor.diffView.applyAll` to `konveyor.applyAll` and registered via `command.ts`